src/primitives/maybeRelocatable.ts

```diff
@@ -0,0 +1,28 @@
+import { Felt } from './felt';
+import { Relocatable } from './relocatable';
+
+export type MaybeRelocatable = Relocatable | Felt;
+
+export function isFelt(
+  maybeRelocatable: MaybeRelocatable
+): maybeRelocatable is Felt;
+export function isFelt(
+  maybeRelocatable: MaybeRelocatable | number
+): maybeRelocatable is Felt;
+export function isFelt(
+  maybeRelocatable: MaybeRelocatable | number
+): maybeRelocatable is Felt {
+  return maybeRelocatable instanceof Felt;
+}
+
+export function isRelocatable(
+  maybeRelocatable: MaybeRelocatable
+): maybeRelocatable is Relocatable;
+export function isRelocatable(
+  maybeRelocatable: number | Relocatable
```

Comment:
set the union type with `number` at the end for consistency with other signatures

```suggestion
  maybeRelocatable: Relocatable | number
```